crates/api/src/community/tag.rs

```diff
@@ -0,0 +1,161 @@
+use activitypub_federation::config::Data;
+use actix_web::web::{Json, Query};
+use lemmy_api_common::{
+  community::{
+    CommunityTagResponse,
+    CreateCommunityTag,
+    DeleteCommunityTag,
+    ListCommunityTags,
+    ListCommunityTagsResponse,
+    UpdateCommunityTag,
+  },
+  context::LemmyContext,
+  utils::check_community_mod_action,
+};
+use lemmy_db_schema::{
+  source::{
+    community::Community,
+    tag::{Tag, TagInsertForm},
+  },
+  traits::Crud,
+};
+use lemmy_db_views::structs::LocalUserView;
+use lemmy_utils::{error::LemmyResult, utils::validation::is_valid_tag_slug};
+use url::Url;
+
+#[tracing::instrument(skip(context))]
```

Comment:
I removed all the instrument macros in https://github.com/LemmyNet/lemmy/pull/5368, they dont serve any purpose.